app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  has_and_belongs_to_many :recipes
+  has_many :recipes, dependent: :destroy
```

Comment:
This is an interesting change! Why did you decide to change the relationship?

Reply:
I don't think we're really utilizing the concept of a recipe having multiple users in the app. It also made the creation of factories (and the necessary associations) for Rspec testing a lot easier, and gives the same functionality as before!

Reply:
That makes a lot of sense!